src/linked-list/doubly-linked-list/DoublyLinkedList.ts

```diff
@@ -17,10 +17,16 @@ export default class DoublyLinkedList {
   private _count: number;
   private _head: DoublyLinkedListNode | null;
   private _tail: DoublyLinkedListNode | null;
+  private _compFunc: (x: any, y: any) => boolean;
+
   constructor() {
```

Comment:
Add a param here, which the user can pass
`constructor(compFunc?: (x: any, y: any) => boolean)`

Reply:
So here, the declaration of `compFunction` doesn't change, as we expect the user to pass a function that takes 2 arguments as input and returns a boolean value. <br>
I will make changes to the constructor.